### PR TITLE
Disable Browser Pay (Donate Now)

### PR DESCRIPTION
**Pull request by mohitb35**
This PR removes Browser Pay (Donate Now), as it was not working correctly (has been deprecated)

### CODE REVIEW

File: src/Utils/browsercheck.ts [hunk 1]
```diff
@@ -34,6 +44,20 @@ function samsung(range: string, userAgent: string) {
...
+  return (
+    safari("<13", userAgent) ||
+    samsung("<9", userAgent) ||
+    !Object.values ||
+    !window.Intl ||
+    !window.crypto
+  );
+}
+
+export function browserPayNotCompatible(): boolean {
```
Comment (norbertschuler) [inline src/Utils/browsercheck.ts:57]: This new function is not used, is it? So I do not understand why this MR is changing this file.

File: src/Donations/PaymentMethods/PaymentRequestCustomButton.tsx [hunk 2]
```diff
@@ -182,22 +181,7 @@ export const PaymentRequestCustomButton = ({
...
-        ) : (
-          <div className="w-100">
-            <button
-              onClick={() => paymentRequest.show()}
-              className={`donate-now ${
-                isPaymentPage ? "donate-small" : "primary-button mb-10 w-100"
-              }`}
-              style={{ border: "none" }}
-            >
-              {isPaymentPage ? "" : t("donateNow")} <BrowserPayIcon />
```
Comment (norbertschuler) [inline src/Donations/PaymentMethods/PaymentRequestCustomButton.tsx:194]: I think the resource `donateNow` is also not used any more after removing this, but I am not sure if we want to keep unused translations.

**Review by norbertschuler — APPROVED** [review]
I checked that the button is shown on https://donate-with-planet-8ct22qzgv-planetapp.vercel.app/?to=1-million-trees&step=donate (resolved from https://paydev.pp.eco/) but no more on https://donate-with-planet-5d8nye446-planetapp.vercel.app/?to=1-million-trees&step=donate in my Chrome browser with some credit card configured.

File: src/Utils/browsercheck.ts (hunk 1 above)
Comment (mohitb35) [reply src/Utils/browsercheck.ts]: This seems to have been left behind after a revert commit. Have removed it.

File: src/Donations/PaymentMethods/PaymentRequestCustomButton.tsx (hunk 2 above)
Comment (mohitb35) [reply src/Donations/PaymentMethods/PaymentRequestCustomButton.tsx:203]: I've removed the unused translations - better to keep things clean.

**Review by norbertschuler — APPROVED** [review]

**Pull request merged**